docs/development/classlib/classlib-manual.rst

```diff
@@ -0,0 +1,72 @@
+.NET Core Class Library Manual
```

Comment:
I'd kind of expected this to just be called ".NET Class Library Manual". Is there a reason we're trying to scope this to whatever ".NET Core" is? As a .NET developer, I just want to know how to build .NET class libraries for **all** of .NET, including this new fangled thing that's called ".NET Core".